doc/api/cli.md

```diff
@@ -346,10 +346,18 @@ Disable experimental support for the [Fetch API][].
 
 <!-- YAML
 added: v16.6.0
- -->
+-->
 
 Use this flag to disable top-level await in REPL.
 
+### `--experimental-shadow-realm`
```

Comment:
Why do we need this flag? `--harmony-shadow-realm` already adds the ShadowRealm global although the ctor throws without the implementation in this PR.

Reply:
Please take a look at https://github.com/nodejs/node/pull/42869/files#diff-5f2fc380364ef43210af4a28c08f9a0c7ec5f85e3ec1f5c1df5e0204eb9dc721R265. We need this flag since `--harmony-shadow-realm` is not a Node.js flag.

Reply:
Ah, right. That reminds me, we should also add an entry for this flag between these two:
https://github.com/nodejs/node/blob/d268cf5a22e8fe4b631a2ce6a484e8d99991fa24/doc/node.1#L156-L160

Reply:
Updated :)